bulbs/poll/api.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework.routers import DefaultRouter
+
+from .viewsets import PollViewSet, AnswerViewSet
+
+router = DefaultRouter(trailing_slash=True)
+
+api_v1_router = DefaultRouter()
+api_v1_router.register(
```

Comment:
No. What's a recommended way to run one? In-editor? Or could we install it to run as part of testing?